samples/charts/financial-chart/tooltip-types/Pages/FinancialChartTooltipTypes.razor

```diff
@@ -0,0 +1,47 @@
+﻿@page "/FinancialChartTooltipTypes"
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@using Samples.Services
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igOptions">
+        <label class="igOptions-label">Tooltip Type: </label>
+        <select @bind="@TooltipType">
+            <option>@ToolTipType.Default</option>
+            <option>@ToolTipType.Item</option>
+            <option>@ToolTipType.Category</option>
+            <option>@ToolTipType.None</option>
+        </select>
+    </div>
+
+    <div class="igComponent">
+
+        @if (DataSource != null)
+        {
+            <FinancialChart Width="100%"
+                Height="100%"
+                ChartType=FinancialChartType.Line
+                Thickness=2
+                YAxisMode="FinancialChartYAxisMode.PercentChange"
+                DataSource="DataSource"
+                ZoomSliderType="FinancialChartZoomSliderType.None"
+                ToolTipType="@TooltipType"/>
+        }
+        
+    </div>
+</div>
+
+@code {
+
+    List<StockItem[]> DataSource;
+    public ToolTipType TooltipType { get; set; } = ToolTipType.Item;
+
+    protected override async Task OnInitializedAsync()
+    {
+        base.OnInitialized();
```

Comment:
you cannot call base's OnInitialized from OnInitializedAsync

Reply:
pending